# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderD3D11TextureHost.cpp
# Commit: 8839a3ba6243
# Full Hash: 8839a3ba62436bd0ea7e5bd07e2b63a8d45b8609
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2023-12-09 09:33:28
# Regressor Bug: 1863914
# File Overlap Count: 2
# Description:
#   Bug 1863914: Use multiple shmem buffers for remote canvas recording. r=aosmond
#   
#   This replaces the use of a single large ring buffer.
#   The buffers are still processed in parallel and are recycled to reduce
#   allocation. Events that do not fit in the default sized buffer have a separate
# ==============================================================================

diff -r f088b7a21045 -r 8839a3ba6243 gfx/webrender_bindings/RenderD3D11TextureHost.cpp
--- a/gfx/webrender_bindings/RenderD3D11TextureHost.cpp	Fri Dec 08 15:56:39 2023 +0000
+++ b/gfx/webrender_bindings/RenderD3D11TextureHost.cpp	Fri Dec 08 16:11:27 2023 +0000
@@ -13,6 +13,7 @@
 #include "RenderCompositorD3D11SWGL.h"
 #include "ScopedGLHelpers.h"
 #include "mozilla/DebugOnly.h"
+#include "mozilla/gfx/CanvasManagerParent.h"
 #include "mozilla/gfx/Logging.h"
 #include "mozilla/layers/GpuProcessD3D11TextureMap.h"
 #include "mozilla/layers/TextureD3D11.h"